### ISSUE TITLE
buttons do not have an accessible name 

### ISSUE THREAD

**Issue by lisagjh** [lisagjh/connect-your-tribe-squad-page#6]
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers. [Learn how to make buttons more accessible](https://dequeuniversity.com/rules/axe/4.8/button-name).

**Comment by lisagjh** [comment]
fixed
